js/piwik.js

```diff
@@ -4032,10 +4032,10 @@ if (typeof window.Piwik !== 'object') {
             /*
              * Log the page view / visit
              */
-            function logPageView(customTitle, customData) {
+            function logPageView(customTitle, customData, callback) {
                 var request = getRequest('action_name=' + encodeWrapper(titleFixup(customTitle || configTitle)), customData, 'log');
 
-                sendRequest(request, configTrackerPause);
+                sendRequest(request, (callback ? 0 : configTrackerPause), callback);
```

Comment:
Thx for sending the PR 👍 

I dont really understand this part: `(callback ? 0 : configTrackerPause)`. Shouldn't we still always send `configTrackerPause` as second parameter?